packages/@atjson/source-wordpress-shortcode/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Just for my own knowledge what does this do?

Reply:
disables the package lock in this package

Reply:
thanks, make sense